### DISCUSSION TITLE
Installation completed!

### DISCUSSION THREAD

**Discussion by AlexandreUSA** [Dilbert66/esphome-vistaECP#46] [Show and tell]
After a couple of days with no issues, I decided to find a long term placement for my VistaAlarm system. I cleaned up the setup, got a breadboard PCB and box, and soldered the components. Note that I am using USB power as I was already close to the 600mA limit of the Vista21ip.

Thanks to @Dilbert66 for his replies to my questions and his super fast bug fixes!

![image](https://user-images.githubusercontent.com/56739736/122678528-4dd4a780-d1b5-11eb-9b00-da9d8b430d76.png)

![image](https://user-images.githubusercontent.com/56739736/122678556-5c22c380-d1b5-11eb-98fe-3269b6beb913.png)

**Comment by appleguru** [comment]
Well done, looks great! What box did you use? I have mine in a cardboard box at the bottom of my alarm panel at the moment…

**Reply by AlexandreUSA** [reply]
Thanks. I used the following PCB/Box combo: [CircuitSetup](https://www.amazon.com/CircuitSetup-Solderable-Breadboard-Arduino-Prototyping/dp/B07H9CV1H9/ref=sr_1_3?dchild=1&keywords=CircuitSetup+Solderable+Breadboard+PCB+for+Electronics&qid=1624295284&sr=8-3) on Amazon.com

**Comment by Dilbert66** [comment]
Nice and tidy!
